Bottom: e0be63f49681ed2522ebb5a955f6c72e2ef3998c
Top:    d283e3ee1bb9101a2cdd074a4db4c02efb852cc2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:36:34 +0000

Refresh of fix-poly_contourincontour-test

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 10b3599..1866a4f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2970,7 +2970,6 @@ poly_ContourInContour (PLINE * poly, PLINE * inner)
   assert (poly != NULL);
   assert (inner != NULL);
   if (cntrbox_inside (inner, poly))
-#if 1
     { /* FIXME: This is SLOW!!
        * Check all points on the contour being tested, because we don't
        * want to falsely return that two contours are inside each other
@@ -2984,9 +2983,6 @@ poly_ContourInContour (PLINE * poly, PLINE * inner)
         } while ((pt = pt->next) != &inner->head);
       return 1;
     }
-#else
-    return poly_InsideContour (poly, inner->head.point);
-#endif
   }
   return 0;
 }
